packages/dotcom-build-code-splitting/package.json

```diff
@@ -21,7 +21,8 @@
     "@financial-times/dotcom-page-kit-cli": "file:../dotcom-page-kit-cli"
   },
   "dependencies": {
-    "get-package-name": "^1.0.0",
+    "disable-tree-shaking-for-chunk-plugin": "^1.1.0",
```

Comment:
![Screenshot 2020-02-07 at 15 29 49](https://user-images.githubusercontent.com/10484515/74042037-b7885300-49be-11ea-80a0-3d5abc17aeed.png)

Reply:
Haha yep, it's based on https://github.com/webpack/webpack/issues/10025